data/json/professions.json

```diff
@@ -396,7 +397,7 @@
     "name": "Sheltered Survivor",
     "description": "At the start of the Cataclysm, you hunkered down in a bomb shelter.  You've spent the past months eating canned food, reading books, and tinkering with stuff in the bunker.  Now it is winter - time to face the world above.",
     "points": 4,
-    "proficiencies": [ "prof_fibers", "prof_knitting" ],
+    "proficiencies": [ "prof_fibers", "prof_knitting", "prof_cooking" ],
```

Comment:
Might cause problems due to lack of previous proficiency, "prof_baking_basic".
Also, weird that sheltered survivor has higher proficiency level than Chef